src/backend/tests/fixtures/get-test-server.ts

```diff
@@ -41,24 +40,18 @@ export const getTestServer = async (
     EMAIL_PASS: "example string",
   };
 
-  const app = next({
-    dev: true,
-  });
-  const handle = app.getRequestHandler();
-
-  let server: Server;
-  await app.prepare().then(() => {
-    server = createServer(async (req, res) => {
-      (req as unknown as { env: Env }).env = env;
-      await handle(req, res);
-    });
-    server.listen(appPort);
+  const server = await builtNextJsFixture({
+    port: appPort,
+    middlewares: [
+      (next) => (req, res) => {
```

Comment:
🐧 Add a comment indicating what is this doing or a link to documentation about how to use this JS fixture.

Reply:
[a1f30cc](https://github.com/sfbrigade/compass/pull/104/commits/a1f30cc165934e58f74f3ba24aad1f97461873d7)